simulation/tests/test_costs.py

```diff
@@ -100,3 +101,40 @@ def test_heat_pumps_are_cheaper_to_reinstall_than_install_first_time(
         reinstall_heat_pump_quote = get_unit_and_install_costs(household, heat_pump)
 
         assert reinstall_heat_pump_quote < new_heat_pump_quote
+
+    @pytest.mark.parametrize("heat_pump", set(HEAT_PUMPS))
+    def test_rhi_annual_payments_are_non_zero_for_households_switching_to_heat_pumps(
+        self, heat_pump
+    ):
+
+        household_with_boiler = household_factory(
+            heating_system=random.choices(list(BOILERS))[0]
+        )
+
+        assert estimate_rhi_annual_payment(household_with_boiler, heat_pump) > 0
+
+    @pytest.mark.parametrize("boiler", set(BOILERS))
+    def test_rhi_annual_payments_zero_for_households_switching_to_boilers(self, boiler):
+
+        household = household_factory(
+            heating_system=random.choices(list(HeatingSystem))[0]
```

Comment:
Doesnt the `household_factory` come pre-populated with a `heating_system` so we don't need to sample randomly from one here for this test?

Reply:
It's just so the test gets exposure to more than than the default `heating_system` in `household_factory()`. I've applied this same principle across a lot of tests.

Reply:
Ah, ok. If we did want to make the `household_factory()` non-deterministic then we could apply the randomness inside that function, to save you from adding this to a lot of separate tests. Happy either way.